### DISCUSSION TITLE
Roller shutters are displayed as blinds

### DISCUSSION THREAD

**Discussion by Peet1985** [SukramJ/aiohomematic#1777] [Q&A] [answered]
Hello,

I have recently had the problem that roller shutters are displayed blinds. Actually not bad because only the fader for the lamella adjustment has been added, but the devices can no longer be controlled easily with Alexa, which my wife is incredibly annoying.
I have already looked in the setting of the HmIP-DRBLI4, here the mode is placed on roller shutters.
I still use a single HmIP-FROLL, which is not the case with it.

What can I do that roller shutters are treated again as a roller shutter?

![grafik](https://github.com/user-attachments/assets/dd86b419-1a8a-42f4-a3be-b912748bfbc3)

![grafik](https://github.com/user-attachments/assets/0d8582e9-a95b-4692-b339-5c18b59be6fa)

**Comment by SukramJ** [comment]
Please ensure that the operational mode of this channel is shutter.

The `value_state` of this entity is `uncertain`
That means that the integration could not fetch data (initially) from the CCU for this device, and has not received events.
This is e.g. the default state after a CCU restart, when the CCU has not yet connected to the device or did not receive any events.

Move the shutter up/down, so that the `value_state` gets `valid`, and restart HA afterwards

**Comment by Peet1985** [comment]
Unfortunately, this does not work, the roller shutter has been risen and driven several times since the last restart of the CCU. HA has also restarted several times in the meantime.
The devices are entered in the CCU and also referenced, so the exact height is output.

**Reply by SukramJ** [reply]
> Please ensure that the operational mode of this channel is shutter.

Hast Du das gemacht?

**Comment by SukramJ** [comment]
Hänge bitte die Gerätedefinition hier an. Diese können mit [export service](https://github.com/danielperna84/custom_homematic#homematicip_localexport_device_definition) erzeugt werden.

Danach[ lösche den Cache](https://github.com/danielperna84/custom_homematic?tab=readme-ov-file#homematicip_localclear_cache).

[Lösche das Gerät aus HA](https://github.com/danielperna84/custom_homematic?tab=readme-ov-file#how-can-a-device-be-removed-from-home-assistant). Durch die Änderung der device_id gehen ggf. Automatisierungen kaputt.

Starte HA neu

**Comment by Peet1985** [comment]
[HmIP-DRBLI4.json](https://github.com/user-attachments/files/17409572/HmIP-DRBLI4.json)


Es hat ja die ganze zeit Funktioniert aber irgendwann dann halt eben nicht mehr.

in der CCU ist es als Rolladen deklariert.
![Screenshot_20241017_090823_Firefox](https://github.com/user-attachments/assets/3d8d2ef7-9ae8-4db2-a5d2-e735f093ec0e)

**Reply by SukramJ** [reply]
Die Gerätebeschreibung besteht aus zwei Dateien.

**Reply by Peet1985** [reply]
Sorry

[HmIP-DRBLI4.json](https://github.com/user-attachments/files/17410097/HmIP-DRBLI4.json)


[HmIP-DRBLI4(1).json](https://github.com/user-attachments/files/17410100/HmIP-DRBLI4.1.json)

**Reply by Peet1985** [reply]
Cache habe ich gelöscht, brachte keinen Erfolg.

Beim Löschen des Gerätes ist da nur das Gerät im Adapter gemeint oder der komplette Adapter?

**Reply by SukramJ** [reply]
Nur aus HA löschen.

**Reply by Peet1985** [reply]
Wie gesagt, ich habe es gelöscht und nach einem Neustart hat er es automatisch wieder hinzugefügt aber mit dem selben Problem.

**Reply by SukramJ** [reply]
Ich denke ich hab den Fehler gefunden.
Seit wann hast Du das Problem?

**Reply by Peet1985** [reply]
seit ca 2 Wochen würde ich sagen, ich weiß es aber nicht genau

**Reply by SukramJ** [reply, marked as answer]
Test bitte mal mit der 1.68.0b0.
Gegebenenfalls must du Vorabversionen für die Integration in HACS aktivieren.

**Reply by Peet1985** [reply]
Habs getestet, läuft. 👍 

Vielen Dank

**Reply by SukramJ** [reply]
Danke für die Rückmeldung

**Comment by Peet1985** [comment]
nach dem löschen des Einzelgeräts & HA-Neustart, keine Änderung

![grafik](https://github.com/user-attachments/assets/d50143df-906e-4b17-af6f-c2c5b88201a6)

**Reply by SukramJ** [reply]
Wie sehen die Attribute aus?

**Reply by Peet1985** [reply]
![grafik](https://github.com/user-attachments/assets/765b6a2d-eb54-43f7-9f5a-f7505256017b)
